libpod/container.go

```diff
@@ -392,8 +392,54 @@ func (c *Container) teardownStorage() error {
 	return nil
 }
 
+// Refresh refreshes the container's state after a restart
+func (c *Container) refresh() error {
```

Comment:
If we move the Init out of kpod create and only do it on kpod run and kpod start we do not need to do any of this.  The only change here would be to clear the libpod database.

Reply:
I'm on the fence here. Just resetting to ContainerStateConfigured is certainly easier on our end, but I feel like it's bad API design - how does CRI-O know when the system has restarted, so it has to redo Init()?

It also doesn't prevent us from moving Init() out of create and into run/start. Just have to ignore the errors saying the container is already initialized.

Reply:
Why do we need to recreate the oci spec file at all?  On reboot the only thing we should need to recreate is any content that was written to the tmpdir correct?  

My goal would be to do nothing with the tmpdir storage until the container is restarted.  The containers would still be created since the oci spec file still exists correct?